fix(ui): run censor on names to filter profanity

kcper opened the pull request:
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?
![image](https://user-images.githubusercontent.com/9114200/129209398-707e2c2a-c3c1-4688-913f-8d369bbcbac0.png)
![image](https://user-images.githubusercontent.com/9114200/129209423-e9436263-9153-424b-b680-03ea27fa9087.png)
![image](https://user-images.githubusercontent.com/9114200/129209542-6e54aeb7-1c28-4949-86b8-4bdab8c56080.png)
![image](https://user-images.githubusercontent.com/9114200/129212968-d288f3cb-5be9-489c-b2d2-810634d17838.png)

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?
Follwup to #119 
### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

Censoring everywhere. Check package .jsons, because my changes wanted to delete 27k lines and I cleaned it manually to leave only relevant stuff.

seiyria commented on frontend/src/components/WeaponIcon.vue:
```diff
@@ -88,6 +88,8 @@ const gColor = new Three.Color(0x413F41);
 const bColor = new Three.Color(0xAF5822);
 const white = new Three.Color(0xFFFFFF);
 
+const censor = new CensorSensor();
```
can this get abstracted to a separate service or somewhere else it can be imported? like the getWeaponNameFromSeed function etc. Then we have one function we can call everywhere instead of initializing this everywhere. If we want to update the profanity we then also have one file we have to do so.

seiyria reviewed (changes requested):
abstract censorSensor

kcper commented:
Added abstract instance. Fixed renames not being loaded in the market, I could see it earlier, because it was my nft and rename was loaded after renaming.
